app/views/log_events/index.html.haml

```diff
@@ -0,0 +1,51 @@
+= content_for :h1 do
+  = text("pages.global_settings")
+
+= content_for :sidebar do
+  = render("admin/shared/sidenav_global", sidenav_tab: "log_events")
+
+%h2= text("pages.event_log")
+
+= form_tag(log_events_path, method: :get) do
+
+  = label_tag(text(:event_filter))
+
+  - option_list = LogEvent::LogEventSearcher::ALLOWED_EVENTS.map { |event| [text(event), event] }
+
+  = select_tag(:events,
+    options_for_select(option_list, params[:events]),
+    multiple: true,
+    class: "js--chosen",
+    data: { placeholder: text(:event_placeholder) })
+
+  = label_tag(text(:start_date))
+  = text_field_tag(:start_date, params[:start_date], class: :datepicker__data)
+
+  = label_tag(text(:end_date))
+  = text_field_tag(:end_date, params[:end_date], class: :datepicker__data)
+
+  = label_tag(text(:query))
```

Comment:
Maybe do the start/end on the right side (we use the date range in a single column on the billing tab) and move the event to the left. I think that will alleviate the spacing problem.